src/routes/v1/mod.rs

```diff
@@ -9,6 +11,23 @@ mod versions;
 pub fn v1_config(cfg: &mut web::ServiceConfig) {
     cfg.service(
         web::scope("api/v1")
+            .wrap_fn(|req, srv| {
+                let current_minute = Utc::now().timestamp() / 60;
+
+                if current_minute % 10 > 5 {
+                    srv.call(req).boxed_local()
+                } else {
+                    async {
+                        Ok(
+                            req.into_response(
+                                HttpResponse::Gone()
+                                    .content_type("application/json")
+                                    .body(r#"{"error":"api_deprecated","description":"API V1 was deprecated several months ago. Please upgrade to V2. Check out https://docs.modrinth.com/docs/migrations/v1-to-v2/"}"#)
```

Comment:
IMO this error is a bit too user-unfriendly. It's less a matter of developers still using it and more a matter of people using outdated applications which use it. Might I suggest the following?
```suggestion
                                    .body(r#"{"error":"api_deprecated","description":"You are using an application that uses an outdated version of Modrinth's API. Please either update it or switch to another application. For developers: https://docs.modrinth.com/docs/migrations/v1-to-v2/"}"#)
```

Reply:
thanks! yeah I agree